inventory/inventoryRepository.js

```diff
@@ -0,0 +1,36 @@
+import { DOES_NOT_EXIST, FAILED_GETTING, INVENTORY, SQLERROR } from '../utils/textConstants.js';
+import { DoesNotExistError } from '../errors/errorTypes/doesNotExistError.js';
+import { FailedGettingError } from '../errors/errorTypes/failedGettingError.js';
+
+export class InventoryRepository {
+  constructor ({ mySQLConnection }) {
+    this.mySQLConnection = mySQLConnection;
+  }
+
+  async getBy (property) {
+    try {
+      const userRows = await this.mySQLConnection.executeQuery(
+        'SELECT userName FROM users WHERE id = (?);',
+        [property]
+      );
+      if (userRows.length === 0) {
+        throw new DoesNotExistError(DOES_NOT_EXIST, INVENTORY);
+      }
```

Comment:
You already have the logic to obtain a user implemented. Get the user from the service and pass it as a filter for the inventory.